docs/user-manual/federation-address.adoc

```diff
@@ -112,96 +111,97 @@ Sample Address Federation setup:
 
 auto-delete::
 For address federation, the downstream dynamically creates a durable queue on the upstream address.
-This is used to mark if the upstream queue should be deleted once downstream disconnects,  and the delay and message count params have been met.
-This is useful if you want to automate the clean up, though you may wish to disable this if you want messages to queued for the downstream when disconnect no matter what.
+This is used to mark if the upstream queue should be deleted once downstream disconnects and the delay and message count params have been met. It is useful in automation of the cleanup. You may wish to disable this, if you want messages to be queued for the downstream when disconnect no matter what.
 
 auto-delete-delay::
-The amount of time in milliseconds after the downstream broker has disconnected before the upstream queue can be eligable for `auto-delete`.
+The amount of time in milliseconds after the downstream broker has disconnected, before the upstream queue can be eligible for `auto-delete`.
 
 auto-delete-message-count::
-The amount number messages in the upstream queue that the message count must be equal or below before the downstream broker has disconnected before the upstream queue can be eligable for `auto-delete`.
+Maximum number of messages allowed in the upstream queue to be eligible for `auto-delete`, after the downstream broker has disconnected.
 
 transformer-ref::
-The ref name for a transformer (see transformer config) that you may wish to configure to transform the message on federation transfer.
+Reference name of a transformer (see transformer config) that you may wish to configure to transform the message on federation transfer.
 
 enable-divert-bindings::
-Setting to true will enable divert bindings to be listened for demand.
-If there is a divert binding with an address that matches the included addresses for the stream, any queue bindings that match the forward address of the divert will create demand.
-Default is `false`.
+Setting to `true` will enable divert bindings to be listened for demand.
+If there is a divert binding with an address that matches the included addresses for the stream, any queue bindings that match the forward address of the divert will create demand. Default is `false`.
 
 NOTE: `address-policy` and `queue-policy` elements can be defined in the same federation and be linked to the same upstream.
 
-Now look at all the `transformer` parameters in turn, in order of priority:
+==== Priority ordered transformer parameters
 
 name::
-This must be a unique name in the server, and is used to ref the transformer in `address-policy` and `queue-policy`
+Unique name in the server used to reference the transformer in `address-policy` and `queue-policy`.
 
 transformer-class-name::
-An optional transformer-class-name can be specified.
+An optional `transformer-class-name` can be specified.
```

Comment:
I dont really see a point in immediately highlighting the name of the attribute, which is already much more significantly bolded / highlighted in the overall attribute list anyway. If changing it, maybe just remove the hyphens instead? Or combine the 'optional' bit into the similar sentence that follows.

Reply:
Agree. I will remove whole sentence. It follows later
`If specified, then the ...`